dependencies.yaml

```diff
@@ -340,7 +340,7 @@ dependencies:
         packages:
           - cachetools
           - cuda-python>=11.7.1,<12.0
-          - numba>=0.56.2
+          - &numba numba>=0.56.4,<0.57
```

Comment:
Probably a basic question, but why does this change into `& numba numba`?

Reply:
This is referred to as a YAML anchor. You can view the spec here: https://yaml.org/spec/1.2.2/#692-node-anchors.

There are some Google results that provide better examples than the official spec docs above.

In this case an anchor was used to keep the `dependencies.yaml` file DRY.